Text/SplitImage/SplitImage.cs

```diff
@@ -0,0 +1,131 @@
+// Copyright (c) Microsoft. All rights reserved.  
+// Licensed under the MIT License. See LICENSE file in the project root for full license information.  
+
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Logging;
+using System.Collections.Generic;
+using AzureCognitiveSearch.PowerSkills.Common;
+using System;
+using Newtonsoft.Json.Linq;
+using System.Net;
+using System.IO;
+using SixLabors.ImageSharp;
+using SixLabors.ImageSharp.Processing;
+using SixLabors.Primitives;
+using SixLabors.ImageSharp.Formats.Jpeg;
+
+namespace AzureCognitiveSearch.PowerSkills.Text.SplitImage
+{
+    public static class SplitImage
+    {
+        private static int MaxImageDimension = 4000; // maximum size of image in cognitive service pipeline
+        private static int ImageOverlapInPixels = 100;
+
+        static SplitImage()
+        {
+            SixLabors.ImageSharp.Configuration.Default.Configure(new TiffLibrary.ImageSharpAdapter.TiffConfigurationModule());
+        }
+
+        [FunctionName("split-image")]
+        public static async Task<IActionResult> RunSplitImageSkill(
+            [HttpTrigger(AuthorizationLevel.Function, "post", Route = null)] HttpRequest req,
+            ILogger log,
+            ExecutionContext executionContext)
+        {
+            log.LogInformation("Split Image Custom Skill: C# HTTP trigger function processed a request.");
+
+            string skillName = executionContext.FunctionName;
+            IEnumerable<WebApiRequestRecord> requestRecords = WebApiSkillHelpers.GetRequestRecords(req);
+            if (requestRecords == null)
+            {
+                return new BadRequestObjectResult($"{skillName} - Invalid request record array.");
+            }
+
+            WebApiSkillResponse response = WebApiSkillHelpers.ProcessRequestRecords(skillName, requestRecords,
+                (inRecord, outRecord) => {
+                    var imageUrl = inRecord.Data["imageLocation"] as string; // no url parameters
+                    var sasToken = inRecord.Data["sasToken"] as string; // includes ? to start query params
+
+                    string fullUri = imageUrl + sasToken;
+                    JArray splitImages = new JArray();
+
+
+                    using (WebClient client = new WebClient())
```

Comment:
Suggestion: use the new using syntax that doesn't require braces, to reduce nesting.

Reply:
projects still on 7.3, going to wont fix

---
In reply to: [368259096](https://github.com/Azure-Samples/azure-search-power-skills/pull/20#discussion_r368259096) [](ancestors = 368259096)